sdk/textanalytics/out/azure/ai/textanalytics/_text_analytics_client.pyi

```diff
@@ -0,0 +1,21 @@
+# Stubs for azure.ai.textanalytics._text_analytics_client (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._base_client import TextAnalyticsClientBase
+from ._credential import TextAnalyticsApiKeyCredential
+from ._generated.models import TextAnalyticsErrorException
+from ._models import AnalyzeSentimentResult, DetectLanguageInput, DetectLanguageResult, DocumentError, ExtractKeyPhrasesResult, RecognizeEntitiesResult, RecognizeLinkedEntitiesResult, RecognizePiiEntitiesResult, TextDocumentInput
+from ._request_handlers import _validate_batch_input
+from ._response_handlers import entities_result, key_phrases_result, language_result, linked_entities_result, pii_entities_result, process_batch_error, sentiment_result
+from azure.core.credentials import TokenCredential
+from typing import Any, Dict, List, Optional, Union
+
+class TextAnalyticsClient(TextAnalyticsClientBase):
+    def __init__(self, endpoint: str, credential: Union[TextAnalyticsApiKeyCredential, TokenCredential], **kwargs: Any) -> None: ...
+    def detect_language(self, inputs: Union[List[str], List[DetectLanguageInput], List[Dict[str, str]]], country_hint: Optional[str]=..., **kwargs: Any) -> List[Union[DetectLanguageResult, DocumentError]]: ...
```

Comment:
The input needs to be homogeneous. If the user passes a List[str], the id for each document gets set internally. If mixed input was allowed, the same id might accidentally get specified twice and cause batch failure.

Reply:
I had a question about the dict format. Are these dicts that users might already be getting as-is from some other service or source? Or are they expected to contract them by hand? If the latter, maybe a helper function `def create_doc_dict(id, lang, text)` would be helpful (and documentable) for creating dicts with the right fields?

Reply:
You you have both a custom "builder" method **and** a model type (`DetectLanguageInput`)? Or is your suggestion to drop the model type and have the builder method?

Reply:
My suggestion is very narrow and weak: If dicts are accepted, and if they do not normally originate pre-made from elsewhere, make it as easy as possible for users to create them correctly. 

I don't have a strong opinion about whether dicts should be accepted at all, because I don't know enough about the users.